test/common/router/config_impl_test.cc

```diff
@@ -6750,6 +6750,69 @@ TEST_F(RoutePropertyTest, TestRouteCorsConfig) {
   EXPECT_EQ(cors_policy->allowCredentials(), true);
```

Comment:
I think you can just add a `EXPECT_EQ(cors_policy->allowPrivateNetworkAccess(), false);` after here, then that is enough.